tests/test_formatter.py

```diff
@@ -675,8 +701,8 @@ def test_keyword_disjoint_comment_stays_keyword_disjoint(self):
         assert formatter.get_formatted() == expected
 
     def test_buffer_with_lone_comment(self):
-        snakecode = 'include: "a"\n# A comment\ninclude: "b"\n'
-        expected = 'include: "a"\n\n\n# A comment\ninclude: "b"\n'
+        snakecode = 'include: "a"\n# A comment\nreport: "b"\n'
+        expected = 'include: "a"\n\n\n# A comment\nreport: "b"\n'
```

Comment:
Don't we want both of these to be treated a single param keywords? i.e. we *don't* want to add the extra newlines?

Reply:
I don't think so: @johanneskoester wants
```
report: "report/workflow.rst"


# Allow users to fix the underlying OS via singularity.
container: "docker://continuumio/miniconda3"


include: "rules/common.smk"
include: "rules/benchmarking.smk"
```

in #67